### ISSUE TITLE
Preloader component

### ISSUE THREAD

**Issue by jonaskuiler** [Blocklevel/blue#39]
Many projects need a preload state to load static files before rendering the site.

@MatteoGabriele maybe there is a way to load the initial language file in this state too

**Comment by MatteoGabriele** [comment]
`Vue.initI18nManager()` loads the default defined language and returns a Promise.
I can also provide a way to load all languages on booting, but I think that is too much. 
Maybe a batch process going on in the background?

**Comment by jonaskuiler** [comment]
The user requests one language when navigating to the site, so the most probable use-case will be that the initial language is incorrect and the user switched to the desired one. So I don't think we need to load them all :).

But it would be really nice if we could load the default language during a preload state.

**Comment by MatteoGabriele** [comment]
Then that's already possible including the Promise of the plugin in the loader

**Comment by jonaskuiler** [comment]
True, but currently the app is not mounted until the locale file is loaded. But I see that you support mounting the app before the promise resolves, so this should be an easy implementation.

**Comment by MatteoGabriele** [comment]
Exactly, that issue is fixed already. The application now is free to render whenever it's possible. 
We only need to avoid the little flickering of text between the label and its translation.
